### PR TITLE
Remove the 'Created' line and display relevant information

### PR DESCRIPTION
**Pull request by lhaskins**
The lines that state "Created <resource>" lines have been removed. In a couple of cases (chat-handles and permissions), the relevant data has been added. All other just displayed the relevant data that was already being displayed.

For permissions, since the relevant information wasn't being displayed, the addition of a View is being used for rendering purposes.

Connected to https://github.com/operable/cog/issues/546

### CODE REVIEW

File: lib/cogctl/actions/groups/create.ex [hunk 1]
```diff
@@ -2,7 +2,6 @@ defmodule Cogctl.Actions.Groups.Create do
   use Cogctl.Action, "groups create"
 
   alias Cogctl.Actions.Groups
-  alias CogApi.HTTP.Client
```
Comment (kevsmith) [inline lib/cogctl/actions/groups/create.ex]: Why modify the alias here?

**Comment by kevsmith** [comment]
Output changes look fine. Can you extract the `Cog.API.Client` changes to a separate PR?

File: lib/cogctl/actions/groups/create.ex (hunk 1 above)
Comment (lhaskins) [inline lib/cogctl/actions/groups/create.ex]: This was just a matter of being consistent across the different files. Not a necessary change.

**Comment by kevsmith** [comment]
👍

**Pull request merged**